app/assets/javascripts/language.js

```diff
@@ -7,7 +7,7 @@ I18n.fallbacks = true;
 I18n.defaultLocale = "nl";
 
 // Language switcher
-const urlParams = new URLSearchParams(window.location.search);
```

Comment:
Why did you change this? Both are constants

Reply:
it gave errors in the console on switching language

Reply:
Hmm weird, okay!